test/features/UpdatePowerShell.test.ts

```diff
@@ -107,6 +107,7 @@ describe("UpdatePowerShell feature", function () {
     });
 
     describe("Which version it gets", function () {
+        this.slow(2000);
```

Comment:
I haven't investigated it, they are probably calling vscode API stuff instead of stubbing them out. slow() re[prts in yellow if the test takes at least 50% of the threshold and red if it takes 100%. This is a potential todo in the future to fix up these tests if they are slow or split them into a separate E2E suite.